gamestonk_terminal/helper_funcs.py

```diff
@@ -932,26 +929,44 @@ def get_rf() -> float:
         return 0.02
 
 
-def try_except(f):
-    """Adds a try except block if the user is not in development mode
+class LineAnnotateDrawer:
+    def __init__(self, ax: matplotlib.axes = None):
+        self.ax = ax
 
-    Parameters
-    -------
-    f: function
-        The function to be wrapped
-    """
-    # pylint: disable=inconsistent-return-statements
-    @functools.wraps(f)
-    def inner(*args, **kwargs):
-        if cfg.DEBUG_MODE:
-            return f(*args, **kwargs)
-        try:
-            return f(*args, **kwargs)
-        except Exception as e:
-            logger.exception("%s", type(e).__name__)
-            return []
-
-    return inner
+    def draw_lines_and_annotate(self):
```

Comment:
You got this repeated.

Also, can you remove that commented code on the function below :)

Reply:
aha